packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -133,9 +140,32 @@ function addEditableCallbacks<T extends DbObject>({collection, options = {}}: {
       const userId = currentUser._id
       const editedAt = new Date()
       const changeMetrics = htmlToChangeMetrics("", html);
+      const isFirstDebatePostComment = 'debate' in doc
+        ? (!!doc.debate && fieldName === 'contents')
```

Comment:
[stupid nit] My confusion on reading that first line is exactly why I prefer ? on the first line of the ternary.

Reply:
ack (but disagree)